src/lexer/token.py

```diff
@@ -1,43 +1,130 @@
+from constants.constants import DELIMS
+from enum import Enum
+
+
+class TokenType(Enum):
+    def __init__(self, token: str, delim_id: str):
+        self._token = token
+        self._delim_id = delim_id
+        self._expected_delims = DELIMS[self.delim_id]
+
+    @property
+    def token(self):
+        return self._token
+
+    @property
+    def delim_id(self):
+        return self._delim_id
+
+    @property
+    def expected_delims(self):
+        return self._expected_delims
+
+    def __str__(self):
+        return self.token
+
+    # GENERAL KEYWORDS
+    START = ("START_KEYWORD", "start_done")
+    DONE = ("DONE_KEYWORD", "start_done")
+    MAINUWU = ("MAINUWU_KEYWORD", "mainuwu")
+    FWUNC = ("FWUNC_KEYWORD", "whitespace")
+    CWASS = ("CWASS_KEYWORD", "whitespace")
+    GWOBAW = ("GWOBAW_KEYWORD", "whitespace")
+    INPUT = ("IWPUT_KEYWORD", "function")
```

Comment:
From docu (specific rules)
```suggestion
    INPWT = ("INPWT_KEYWORD", "function")
```

But from regex, `inpuwut` siya, not sure kung ano finall